src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -124,8 +124,51 @@ def set_package_assessment_status(self, package_names, package_versions, classif
                 }
                 self.__assessment_packages.append(record)
 
+        self.__assessment_packages = self.sort_packages_by_classification_and_state(self.__assessment_packages)
         self.set_assessment_substatus_json()
 
+    def sort_packages_by_classification_and_state(self, packages_list):
+        """ Sorts a list of packages (usually either self.__assessment_packages or self.__installation_packages) by classification and patchState properties.
+            (sorting order from highest priority to lowest):
+            1. Classification: Security, Critical, Other, Unclassified
+            2. Patch Installation State: Failed, Installed, Available, Pending, Excluded, NotSelected
+        """
+        classification_order = {
+            Constants.PackageClassification.SECURITY: 1,
+            Constants.PackageClassification.CRITICAL: 2,
+            Constants.PackageClassification.OTHER: 3,
+            Constants.PackageClassification.UNCLASSIFIED: 4
+        }
+
+        patch_state_order = {
+            Constants.FAILED: 1,
+            Constants.INSTALLED: 2,
+            Constants.AVAILABLE: 3,
+            Constants.PENDING: 4,
+            Constants.EXCLUDED: 5,
+            Constants.NOT_SELECTED: 6
+        }
```

Comment:
These 2 can be constants instead of a function

Reply:
Done